Linear_cell_complex/demo/Linear_cell_complex/MainWindow.ui

```diff
@@ -38,7 +38,7 @@
    </property>
    <widget class="QMenu" name="menuFile">
     <property name="title">
-     <string>&amp;File</string>
```

Comment:
Because I have a conflict with one menu shortcut and one key used by the basic viewer. 
I can try to change the char used for the shortcut menu... but I think this is nor really important the lcc demo.